include/nds/libversion.h

```diff
@@ -5,10 +5,18 @@
 #ifndef LIBNDS_NDS_LIBVERSION_H__
 #define LIBNDS_NDS_LIBVERSION_H__
 
+#define _BLOCKSDS_MAJOR_ 1
+#define _BLOCKSDS_MINOR_ 3
+#define _BLOCKSDS_PATCH_ 0
+
+#define _BLOCKSDS_STRICT_CURRENT_ (((_BLOCKSDS_MAJOR_) * 10000) + ((_BLOCKSDS_MINOR_) * 100) + (_BLOCKSDS_PATCH_))
+
+// The below defines mark the version of libnds with which BlocksDS tries to be compatible.
+
 #define _LIBNDS_MAJOR_ 1
 #define _LIBNDS_MINOR_ 8
-#define _LIBNDS_PATCH_ 0
+#define _LIBNDS_PATCH_ 2
 
-#define _LIBNDS_STRING "libnds release 1.8.0"
+#define _LIBNDS_STRING "BlocksDS " _BLOCKSDS_MAJOR_ "." _BLOCKSDS_MINOR_ "." _BLOCKSDS_PATCH_ " (like libnds " _LIBNDS_MAJOR_ "." _LIBNDS_MINOR_ "." _LIBNDS_PATCH_ ")"
```

Comment:
I'm not sure about this. If we're dropping compatibility with old stuff, we might as well drop the old version number. Also, if we wanted to do the same in libraries such as DSWiFi we would have two sets of `_BLOCKSDS_xxxx_`.

Reply:
It's provided for compatibility reason - the idea, as we discussed, is that we're (mostly) keeping compatibility with libnds, no? Only when `_BLOCKSDS_STRICT_` is defined do we start breaking things en masse.

As for the other libraries: maxmod isn't likely to see major changes (its API is fairly clean and featureset largely complete), while DSWiFi has its own `dswifi_version.h` file and a much smaller API surface.

Reply:
To clarify:

- I kept the integer version number so that defines which try to detect the presence of a specific libnds version continue working as expected,
- I changed the string mostly so that if someone uses it to print the version of libnds used by their code, it will be easy to distinguish between devkitARM's upstream and our own builds.